content/blog/2020-06-08-june-20-heartbeat.md

```diff
@@ -0,0 +1,213 @@
+it out if you missed the event.
+
+https://youtu.be/19GMtrFykSU
+
+## From the community
+
+As usual, there's a ton of noteworthy action in the DVC community.
+
+[Derek Haynes](https://twitter.com/dhaynes23), MLOps expert and new
+[DVC Ambassador](https://dvc.org/blog/dvc-ambassador-program-announcement)-
+wrote an excellent overview of using
+[GitHub CodeSpaces](https://github.com/features/codespaces/). CodeSpaces is a
+new development environment (currently in beta) that we're eagerly watching. As
+Derek shows in his blog, it lets you have a Jupyter Notebook experience without
+sacrificing on development standards- he uses
+[whisk](https://docs.whisk-ml.org/en/latest/) to structure the project and
+manage Python package dependencies, and DVC to version the model training
+pipeline.
+
+This use case is telling in the
+[battle over Jupyter notebooks](https://towardsdatascience.com/the-case-against-the-jupyter-notebook-d4da17e97243):
+we might just be able to have both a notebook _and_ mature project management.
+Give Derek's blog a read and tell us what you think.
+
+<external-link
+href="https://dlite.cc/2020/05/26/github-codespaces-machine-learning.html"
+title="GitHub Codespaces for Machine Learning"
+description="With Codespaces, contributors can spin up a ready-to-go GitHub project-specific dev environment in the cloud. In this post, I’ll show how to give potential contributors a graceful start by configuring Codespaces for an ML project."
+link="dlite.cc"
+image="/uploads/images/2020-06-08/derek_haynes.jpg"/>
+
+DVC Ambassador Marcel gave a tutorial about DVC to a bioinformatics student
+group, and then an even bigger talk at the Federal University of Rio Grande de
+Norte. His talk focused on how to use DVC in the context of scientific
+reproducibility- specifically, large biological datasets, which are often
+transformed and processed several times before ML models are fit. In my
+experience, Git-flow is severely underutilized in life sciences research, so
+it's exciting to see Marcel's ideas getting a big audience.
+
+https://twitter.com/ppgeecufrn/status/1263260554443005954
+
+Also, Marcel is the first author of a new scientific paper about mobility data
+across 131 countries during the COVID-19 pandemic. The preprocessing pipeline is
+versioned with DVC. We don't know how Marcel gets this much done.
+
+<external-link
+href="https://www.sciencedirect.com/science/article/pii/S2352340920305928"
+title="Dataset for country profile and mobility analysis in the assessment of COVID-19 pandemic"
+description="M. Ribeiro-Dantas, G. Alves, R.B. Gomes, L.C.T. Bezerra, L. Lima and I. Silva"
+link="sciencedirect.com"
+image="/uploads/images/2020-06-08/data_in_brief_logo.jpeg"/>
+
+Also just released is a scientific paper by Christoph Jansen et al. about a
+framework for computational reproducibility in the life sciences that integrates
+DVC. The framework is called Curious Containers- definitely worth checking out
```

Comment:
we can put a link to the `Curious Containers` GH

Reply:
done!